phoneApp/src/js/view.js

```diff
@@ -1,6 +1,59 @@
 class View{
```

Comment:
it's not required to name it **View**. it could be a "mental abstraction" in your head.

On complex projects, MVC could be hidden, but from the relationships between classes/functions you would feel MVC :) 
"it's smell like MVC"